index.html

```diff
@@ -1,16 +1,22 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="author" content="digitsensitive">
-    <title>phaser3-typescript</title>
-    <link rel="stylesheet" type="text/css" href="styles/css/styles.css">
-    <script src="build/bundle.js"></script>
-  </head>
-  <body>
-    <div class="container">
-      <h1>phaser3-typescript</h1>
-      <div id="game"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="author" content="digitsensitive">
+        <title>TETRIS</title>
+        <!-- <link rel="stylesheet" type="text/css" href="styles/css/styles.css"> -->
+        <link rel="stylesheet" type="text/css" href="assets/css/main.css">
+        <link rel="stylesheet" type="text/css" href="assets/css/fontLoader.css">
+        <script src="build/bundle.js"></script>
+    </head>
+    <body>
+        <div class="container">
+            <center>
+                    <div id="game"></div>
```

Comment:
We should use css to center the container.
e.g.:
```
div#game {
    margin: 0 auto;
}
```

Reply:
```margin: auto``` on its own didn't work, but I added ```text-align: center``` and now it's centered.